sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -218,49 +240,84 @@ fetched_config_setting = client.get_configuration_setting(key="MyKey", label="My
 
 Delete an existing Configuration Setting.
 
-<!-- SNIPPET:hello_world_advanced_sample.delete_config_setting -->
+<!-- SNIPPET:hello_world_sample.delete_config_setting -->
 
 ```python
-client.delete_configuration_setting(
-    key="MyKey",
-    label="MyLabel",
-)
+client.delete_configuration_setting(key="MyKey", label="MyLabel")
 ```
 
 <!-- END SNIPPET -->
 
 ### List Configuration Settings
 
-List all configuration settings filtered with label_filter and/or key_filter.
+List all configuration settings filtered with label_filter and/or key_filter and/or tags_filter.
+
+<!-- SNIPPET:list_configuration_settings_sample.list_configuration_settings -->
+
+```python
+config_settings = client.list_configuration_settings(key_filter="MyKey*", tags_filter=["my tag1=my tag1 value"])
+for config_setting in config_settings:
+    print(config_setting)
+```
+
+<!-- END SNIPPET -->
+
+### List revisions
+
+List revision history of configuration settings filtered with label_filter and/or key_filter and/or tags_filter.
+
+<!-- SNIPPET:list_revision_sample.list_revisions -->
+
+```python
+items = client.list_revisions(key_filter="MyKey", tags_filter=["my tag=my tag value"])
```

Comment:
After seeing this example - I'm wondering if `tag_filters` is actually a better name....

Reply:
We decided to use the name `tags_filter` after talking offline: 1) it's the name used in API and other languages. 2) it's one filter which consist of a list of tag name-value pairs, not several filters.